io.welle.welle-gui.yml

```diff
@@ -0,0 +1,103 @@
+app-id: io.welle.welle-gui
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: welle-io
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--device=all'
```

Comment:
`all` includes `dri`, but why do you need `all`?

Reply:
welle.io needs access via `libusb` to USB dongles. During my tests, I had issues accessing USB. With `all` it was working. But maybe I miss something.

Reply:
I did a test, with `--device=usb`. `flatpak` gives me the error `error: Unknown device type usb, valid types are: dri, all, kvm, shm`

I'm using Ubuntu 24.04 for development
```
$ flatpak --version
Flatpak 1.14.6
```

I think I have to stay with `--device=all`

Removing `--device=dri` done in https://github.com/AlbrechtL/flathub/commit/53e6d65eae558e47e1c468b8ed4b67c18881efc2.